src/fsharp/DotNetFrameworkDependencies.fs

```diff
@@ -34,6 +34,98 @@ module internal FSharp.Compiler.DotNetFrameworkDependencies
                     None
         with _ -> None
 
+    // Compare nuget version strings
+    //
+    // Format:
+    // =======
+    //      $(Major).$(Minor).$(Build) [-SomePrefix]
```

Comment:
nit: suffix, not prefix

Reply:
How awesome that the code uses suffix, it's just my useless comment that got it wrong.